packages/sdk/src/chains/substrate.ts

```diff
@@ -248,6 +248,27 @@ export class SubstrateChain implements IChain {
 					},
 				]),
 			)
+			.with({ kind: "GetRequest" }, (message) =>
+				Vector(Message).enc([
+					{
+						tag: "GetRequestMessage",
+						value: {
+							requests: message.requests.map((r) => convertIGetRequestToCodec(r)),
+							proof: {
+								height: {
+									height: message.proof.height,
+									id: {
+										consensusStateId: Array.from(toBytes(message.proof.consensusStateId)),
+										id: convertStateMachineIdToEnum(message.proof.stateMachine) as any,
+									},
+								},
+								proof: Array.from(hexToBytes(message.proof.proof)),
+							},
+							signer: Array.from(hexToBytes(message.signer)),
+						},
+					},
+				]),
+			)
```

Comment:
Substrate chains don't yet support GetRequests. So lets return an error from this branch